### ISSUE TITLE
Question about Training Code

### ISSUE THREAD

**Issue by lh-zhu** [UX-Decoder/FIND#3]
Greetings!

I wish to express my appreciation for your excellent work. I am highly interested in it. I am curious as to whether you could kindly share the training code for the purpose of reproduction. Thank you immensely. I look forward to receiving your early reply.

**Comment by MaureenZOU** [comment]
I believe this is tested under the code base, I will shortly add it to the read me, thanks!
CUDA_VISIBLE_DEVICES=0 python entry.py train \
            --conf_files configs/grin/focalt_llama_lang.yaml \
            --overrides \
            MODEL.DECODER.HIDDEN_DIM 512 \
            MODEL.ENCODER.CONVS_DIM 512 \
            MODEL.ENCODER.MASK_DIM 512 \
            MODEL.DECODER.NUM_OBJECT_QUERIES 101 \
            FP16 True \
            MODEL.DECODER.MASK.ENABLED True \
            MODEL.DECODER.CAPTION.ENABLED True \
            MODEL.DECODER.SPATIAL.ENABLED True \
            MODEL.DECODER.RETRIEVAL.ENABLED True \
            MODEL.DECODER.GROUNDING.ENABLED True \
            MODEL.DECODER.INTERLEAVE.ENABLED True \
            MODEL.DECODER.INTERLEAVE.VISUAL_PROB 0.5 \
            SOLVER.MAX_NUM_EPOCHS 50 \
            SOLVER.BASE_LR 0.0001 \
            SOLVER.FIX_PARAM.backbone True \
            SOLVER.FIX_PARAM.lang_encoder True \
            SOLVER.FIX_PARAM.pixel_decoder True \
            COCO.TEST.BATCH_SIZE_TOTAL 1 \
            COCO.TRAIN.BATCH_SIZE_TOTAL 2 \
            COCO.TRAIN.BATCH_SIZE_PER_GPU 2 \
            VLP.TEST.BATCH_SIZE_TOTAL 1 \
            VLP.INPUT.SHORTEST_EDGE True \
            VLP.INPUT.MIN_SIZE_TEST 512 \
            VLP.INPUT.MAX_SIZE_TEST 864 \
            ADE20K.TEST.BATCH_SIZE_TOTAL 1 \
            REF.TEST.BATCH_SIZE_TOTAL 1 \
            VOC.TEST.BATCH_SIZE_TOTAL 1 \
            FIND_UNUSED_PARAMETERS True \
            COCO.INPUT.IMAGE_SIZE 640 \
            COCO.INPUT.MIN_SIZE_TRAIN 512 \
            COCO.INPUT.MAX_SIZE_TRAIN 864 \
            COCO.INPUT.MIN_SIZE_TEST 512 \
            COCO.INPUT.MAX_SIZE_TEST 864 \
            VOC.INPUT.MIN_SIZE_TEST 512 \
            VOC.INPUT.MAX_SIZE_TEST 864 \
            MODEL.DECODER.SPATIAL.MAX_ITER 0 \
            MODEL.DECODER.TOP_GROUNDING_LAYERS 6 \
            MODEL.DECODER.TOP_SPATIAL_LAYERS 6 \
            MODEL.DECODER.TOP_RETRIEVAL_LAYERS 6 \
            MODEL.DECODER.TOP_INTERLEAVE_LAYERS 6 \
            MODEL.TEXT.LOAD_TENSOR False \
            WEIGHT True \
            FAKE_UPDATE False \
            RESUME_FROM /disk1/checkpoint/vlcore/xdecoder/vision_xdecoder_focalt_unicl_fb_lang_grin_v1/default/model_state_dict.pt
            # MODEL.TEXT.PRECOMPUTE True \
            # COCO.INPUT.DATASET_MAPPER_NAME coco_interleave_language \
            # RESUME_FROM /nobackup3/xueyan-data/grin_data/output/xdecoder/vision_xdecoder_davitd3_unicl_fb_lang_grin_v1/default/model_state_dict.pt
            # RESUME_FROM /nobackup3/xueyan-data/grin_data/output/xdecoder/vision_xdecoder_davitd5_unicl_fb_lang_grin_v1/default/model_state_dict.pt
